package-lock.json

```diff
@@ -18594,6 +18705,75 @@
         }
       }
     },
+    "inquirer-table-prompt": {
```

Comment:
@aman19K The `inquirer-table-prompt` was removed from the utility. I'm not sure how still it's showing up. we need to fix it.